tests/sentry/receivers/test_superuser.py

```diff
@@ -0,0 +1,53 @@
+from sentry.auth.superuser import is_active_superuser
+from sentry.models import User
+from sentry.receivers.superuser import disable_superuser, enable_superuser
+from sentry.testutils import TestCase
+
+
+class SuperuserReceiverTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.superuser = User(is_superuser=True, email="test@sentry.io")
+        self.non_superuser = User(is_superuser=False, email="test@sentry.io")
+
+        self.superuser_request = self.make_request(user=self.superuser)
```

Comment:
Does `make_request` return a 'request'? Or does it actually trigger the request and return the user object?

Reply:
it will just return a httprequest object

Reply:
🤦 uggh. naming sigh...

Wish this was `make_request_object` lol